traitsui/examples/demo/Extras/tests/test_Image_editor_demo.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
The emptiness of at the top of the file is a bit disturbing... It needs a docstring, and possibly a copyright header.